src/components/navs/AppNav/AppNavActivityBtn/ActivityRows.vue

```diff
@@ -22,10 +22,13 @@
         </div>
       </BalLink>
       <div>
-        <CheckIcon
-          v-if="transaction.status === 'confirmed'"
-          class="text-green-500"
-        />
+        <template v-if="transaction.status === 'confirmed'">
+          <CheckIcon
+            v-if="isSuccessfulTransaction(transaction)"
+            class="text-green-500"
+          />
+          <BalIcon v-else name="alert-circle" class="text-red-500" />
```

Comment:
This alert icon might change - kinda a placeholder for now. This is just an example of how different status types may look... so for example a transaction mail fail (revert), and altho it's "confirmed" (mined) it isn't really successful - so it was important to show that info.